account_move_line_product/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# © 2016-17 Eficent Business and IT Consulting Services, S.L.
```

Comment:
Could you change the copyright to:

`Copyright 2019 ForgeFlow S.L. (https://www.forgeflow.com)`

Thanks in advance! :)

Reply:
Changed.